.changeset/wild-goats-march.md

```diff
@@ -0,0 +1,20 @@
+---
+'@itwin/itwinui-react': minor
+'@itwin/itwinui-css': minor
+---
+
+Added a new FileUploadCard component which serves as a default UI for when a single file is uploaded. This can also be used as a child of FileUpload
+
+```jsx
+<FileUploadCard />
+```
+
+```jsx
+const [files, setFiles] = React.useState<File[]>([]);
+
+<FileUpload
+  onFileDropped={console.log()}
+>
+  <FileUploadCard data={files} onDataChange={(files) => setFiles(files)} />
```

Comment:
update this example to use correct prop names

Reply:
This has been changed